eq-author/cypress/builders/answer.js

```diff
@@ -1,23 +1,50 @@
 import { testId } from "../utils";
 
-const configureCheckbox = ({ options }) => {
+const configureMultipleChoice = initialCount => ({ options }) => {
   options.forEach(({ label }, index) => {
-    cy.get(testId("option-label")).should("have.length", index + 1);
+    cy.get(testId("option-label")).should(
+      "have.length",
+      Math.max(index + 1, initialCount)
+    );
     cy.get(testId("option-label"))
-      .last()
+      .eq(index)
       .type(label)
       .blur();
-    const isLast = index + 1 === options.length;
-    if (!isLast) {
+    const nextIndex = index + 1;
+    const shouldAddOne =
+      initialCount <= nextIndex && options.length > nextIndex; // index + 1 === options.length;
+    if (shouldAddOne) {
       cy.get(testId("btn-add-option")).click();
     }
   });
 };
 
+const configueBasicAnswer = ({ label, description }) => {
+  if (label) {
+    cy.get(testId("txt-answer-label")).type(label);
+  }
+  if (description) {
+    cy.get(testId("txt-answer-description")).then(
+      ele =>
```

Comment:
Why `ele`?

Reply:
Just the name I gave to the Jquery element the `cy.get` returns, got a clearer name?

Reply:
`$el` was the jQuery convention IIRC